src/lib/twizzler-abi/src/device/mod.rs

```diff
@@ -112,6 +115,15 @@ bitflags::bitflags! {
 #[repr(transparent)]
 pub struct InterruptVector(u32);
 
+impl TryFrom<u64> for InterruptVector {
```

Comment:
I am baffled by what this code is for.  Can you add a brief, block, comment here?

Reply:
When the kernel returns information about an interrupt vector, it's in a 64 bit value (well, register size, anyway) so this allows conversion from that to the InterruptVector newtype (but, of course, it can fail).